.github/dependabot.yml

```diff
@@ -67,3 +67,24 @@ updates:
     commit-message:
       prefix: "serverless-bench"
       include: "scope"
+
+  - package-ecosystem: "pip"
+    directory: "/components/runtimes/python39/kubeless"
+    labels:
+      - "area/dependency"
+      - "kind/chore"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "pip-python39"
+      include: "scope"
+  - package-ecosystem: "pip"
```

Comment:
I dont think `/tests/serverless-bench` is executed anymore.
Please skip it in dependabot configuration

Reply:
Should I remove it from Docker config as well?